### ISSUE TITLE
Colab error in Inference [Pytorch3D error].

### ISSUE THREAD

**Issue by cduguet** [facebookresearch/meshrcnn#90]
I was getting this error when executing the colab notebook in the [inference cell](https://colab.research.google.com/drive/1eQLZrNYRZMo9zdnGGccE0hFswGiinO-Z#scrollTo=ML4EWtAbzeZG&line=1&uniqifier=1).

```
** fvcore version of PathManager will be deprecated soon. **
** Please migrate to the version in iopath repo. **
https://github.com/facebookresearch/iopath 

Traceback (most recent call last):
  File "./meshrcnn/demo/demo.py", line 15, in <module>
    from pytorch3d.io import save_obj
  File "/usr/local/lib/python3.6/dist-packages/pytorch3d/io/__init__.py", line 4, in <module>
    from .obj_io import load_obj, load_objs_as_meshes, save_obj
  File "/usr/local/lib/python3.6/dist-packages/pytorch3d/io/obj_io.py", line 14, in <module>
    from pytorch3d.renderer import TexturesAtlas, TexturesUV
  File "/usr/local/lib/python3.6/dist-packages/pytorch3d/renderer/__init__.py", line 3, in <module>
    from .blending import (
  File "/usr/local/lib/python3.6/dist-packages/pytorch3d/renderer/blending.py", line 9, in <module>
    from pytorch3d import _C
ImportError: /usr/local/lib/python3.6/dist-packages/pytorch3d/_C.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZNK2at6Tensor7is_cudaEv
```
This looks like a dependency issue, so I'm currently trying the [solution proposed by Babilinski](https://github.com/facebookresearch/meshrcnn/issues/88#issuecomment-751281293).  I'm still not sure if I'm in the right direction solving it, or it might be something else.
Any suggestions, @albertotono ? 

Thank you very much!!

**Comment by mahavirbha** [comment]
ModuleNotFoundError: No module named 'detectron2.utils.file_io'
![image](https://user-images.githubusercontent.com/70435819/110581236-26227c00-8190-11eb-9276-545f8f71ab21.png)

**Comment by ddxue** [comment]
I fixed the above issues by fixing fvcore to **0.1.3.post20210317** and the Detectron code to commit `4b539e41f4b19570bc6b722fa0f516f927cdae02`. See this related issue (https://github.com/facebookresearch/detectron2/issues/2795) where `iopath` made a breaking change in 0.1.7.

```# install dependencies: (use cu101 because colab has CUDA 10.1)
%env FORCE_CUDA=1
!pip install -U torch==1.6 torchvision==0.7 -f https://download.pytorch.org/whl/cu101/torch_stable.html 
!pip install pyyaml==5.1 pycocotools>=2.0.1
!pip install -U fvcore==0.1.3.post20210317
!pip install -U --force-reinstall iopath==0.1.6 
!pip install 'git+https://github.com/facebookresearch/detectron2.git@4b539e41f4b19570bc6b722fa0f516f927cdae02'
!pip install 'git+https://github.com/facebookresearch/pytorch3d.git'
